modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/jaxrs/container/request/filter/TransactionContainerRequestFilter.java

```diff
@@ -138,6 +138,10 @@ private static class TransactionCleanUpMessageObserver
 		extends AbstractPhaseInterceptor implements MessageObserver {
 
 		public void commit() {
+			if (_transactionStatusAdapter.isCompleted()) {
```

Comment:
hey @holatuwol ,

Just a question, which is the difference between using the boolean private attribute `_complete` and `_transactionStatusAdapter.isCompleted()`? Could it be simplified by using just one?

I am trying to reproduce the error but by now, I can't with the log steps

Reply:
Yes, we could just use `_transactionStatusAdapter.isCompleted()`, I'll push an update.

Reply:
@ccorreagg Pushed the requested change to the branch.

Reply:
Thank you @holatuwol